symengine/python/cmake/cython_test.pyx

```diff
@@ -83,3 +83,8 @@ cdef extern from "<symengine/add.h>" namespace "SymEngine":
 
     cdef cppclass Add(Basic):
         void as_two_terms(const Ptr[RCP[Basic]] &a, const Ptr[RCP[Basic]] &b)
+
+
+# Test that cpdef void is supported (https://github.com/cython/cython/commit/7df9d654b3e6c3d5812a95904872d35a3c94ef77)
```

Comment:
That's a really recent commit. Do we really need this feature? This make SymEngine a lot harder to install, as you found out yourself.